src/wrappers/tasks.ts

```diff
@@ -14,26 +14,35 @@ const addDefaultsToAll = (tasks: Task[]): ITask[] =>
 
 export default class {
   private service: TasksApi
+  private authService: AuthorizationsApi
```

Comment:
does this need to be initialized in the constructor?

Reply:
I updated it to initialize the authService in the constructor.